interpretation/ww_head.py

```diff
@@ -871,8 +871,9 @@ def process(message, isGameMaster=False, isAdmin=False):
         # TODO
         return todo()
     if is_command(message, ['profile'], True):
-        # TODO
-        return todo()
+        msg = "**USAGE:** The use of this command is so that you can view your own profile, if you add another players name, you can view their profile . /n/n` + prefix + /n/n **Example:** `!profile or !profile Randium"
```

Comment:
The `/n` should be `\n`. Also there is a space after the word "Profile"

Reply:
This one still has /n instead of \n

Reply:
again, that has been fixed in the lates pull request!

Reply:
You mean that you created a different pull request? That's kinda messy. Why didn't you alter this pull request?

Reply:
cause i made that pull request before knowing this problem existed, but before even knowing this was a problem, i changed it cause i noticed it was different to the rest